Date: Sep 12 15:15:31 2019

From: 1460800cf0cbf969a4d48511beca480fcb6ccf4c Sep 13 02:29:42 2019

Subject: [PATCH] SKYH4-1479: Yocto recipe for tinyproxy

	Include tinyproxy recipe.
	Tinyproxy is used in HUB4 for the selfheal functionality. As tinyproxy
	recipe is not available from daisy, we have ported it.

        xfsprogs: drop configure check for aio
        It's unused and breaks compilation with uclibc.

Source: OPENEMBEDDED


Upstream-Status: Submitted [ http://layers.openembedded.org/layerindex/recipe/63463/ ]

Signed-off-by: Benjamin Gaignard <benjamin.gaignard@linaro.org>
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Signed-off-by: Joe MacDonald <joe_macdonald@mentor.com>
---

--- a/configure.ac	2016-01-01 17:42:27.000000000 +0100
+++ b/configure.ac	2017-04-20 14:03:37.238254471 +0200
@@ -284,24 +284,12 @@
 AC_SUBST(LIBS)
 AC_SUBST(ADDITIONAL_OBJECTS)
 
-# Check for asciidoc
-AC_PATH_PROG(A2X, a2x, no)
-AM_CONDITIONAL(HAVE_A2X, test "x$A2X" != "xno")
-if test x"$A2X" = x"no"; then
-  AC_MSG_ERROR([Test for asciidoc failed. See the file 'INSTALL' for help.])
-fi
-
 AC_CONFIG_FILES([
 Makefile
 src/Makefile
 data/Makefile
 data/templates/Makefile
 etc/Makefile
-docs/Makefile
-docs/man5/Makefile
-docs/man5/tinyproxy.conf.txt
-docs/man8/Makefile
-docs/man8/tinyproxy.txt
 m4macros/Makefile
 tests/Makefile
 tests/scripts/Makefile
--- a/Makefile.am	2016-01-01 17:42:27.000000000 +0100
+++ b/Makefile.am	2017-04-20 14:03:53.842822997 +0200
@@ -2,7 +2,6 @@
 	src \
 	data \
 	etc \
-	docs \
 	m4macros \
 	tests
